images/router/haproxy/conf/haproxy-config.template

```diff
@@ -131,7 +131,8 @@ defaults
   # backend service or alternatively, you can send a custom 503 error.
   #
   # server openshift_backend 127.0.0.1:8080
-  errorfile 503 /var/lib/haproxy/conf/error-page-503.http
+  errorfile 503 {{ env "ERRORFILE_503" "/var/lib/haproxy/conf/error-page-503.http" }}
```

Comment:
I suggest naming the environment variable ROUTER_ERRORFILE_503/404, as seems to be a standard for the other environment variables to be prefixed with ROUTER